harness/determined/common/experimental/determined.py

```diff
@@ -64,6 +64,19 @@ def __init__(
 
         self._session = api.Session(master, user, auth, cert)
 
+    def create_user(username: str, password: str, admin: bool):
+        # call API create_user
+        # create User object
+        # return that object 
+        pass
+    
+    def get_user():
```

Comment:
would be great to get a user by user_id or by username - i.e. support both because we do both often.